codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2Connection.java

```diff
@@ -296,6 +296,7 @@ final DefaultPropertyKey verifyKey(PropertyKey key) {
         private int totalChildWeights;
         private int prioritizableForTree = 1;
         private boolean resetSent;
+        private boolean headerSent;
```

Comment:
Discussed with @nmittler offline. We will re-analyze the design as part of https://github.com/netty/netty/issues/4489.